### PR TITLE
[CLN] website_sale(_*): tests

### PR DESCRIPTION
**Pull request by Feyensv**
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale/tests/test_customize.py [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
 from odoo.tools.misc import file_open
 
+from odoo.addons.base.tests.common import HttpCaseWithUserDemo, HttpCaseWithUserPortal
+
 
 @tagged('post_install', '-at_install')
-class TestUi(HttpCaseWithUserDemo, HttpCaseWithUserPortal):
+class TestCustomize(HttpCaseWithUserDemo, HttpCaseWithUserPortal):
 
     def setUp(self):
```
Comment (Demesmaeker) [inline addons/website_sale/tests/test_customize.py:15]: setUpClass? Didn't check all the files, but I think it could be changed in more than just this one

**Review by Demesmaeker — COMMENTED** [review]
Didn't read every file yet, but these commits lgtm.

You asked for suggestions, so not on this PR that's nice on it's own, but these are some minor nitpickings clean that we could still do:
- it could be nice to rename some of these tests too. `test_00_delivery_cost` isn't really giving away what it is testing for instance. Also, do we really want to continue with numbering the tests? I thought we stopped doing it.
- some comments like: `# I add delivery cost in Sales order` could be removed/reworded to make thing clearer instead of heavier.
- (this one you asked to be remembered xD 
image
 )

Not nitpicking:
- pricelist coverage is incomplete at best in these tests. Or magical. For instance, in `test_customize`, we set the pricelist currency... while there might not have a pricelist at all from what I read (didn't check to be 100% sure).
- You took off the demo data used for the tours in `website_sale_picking`, but all the tours of `website_sale` also depends on these demo data. There could be a task dedicated just to that, though.

I could probably think on more, but let's already go on with a first cleaning ^^

File: addons/website_sale_picking/models/payment_provider.py [hunk 2]
```diff
@@ -11,7 +13,7 @@ class PaymentProvider(models.Model):
     )
 
     @api.model
-    def _get_compatible_providers(self, *args, sale_order_id=None, website_id=None, **kwargs):
+    def _get_compatible_providers(self, company_id, *args, sale_order_id=None, website_id=None, **kwargs):
         """ Override of payment to exclude onsite providers if the delivery doesn't match.
```
Comment (AntoineVDV) [inline addons/website_sale_picking/models/payment_provider.py:18]: missing arg

**Comment by Feyensv** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**